packages/unlock-js/src/Unlock/v10/index.js

```diff
@@ -0,0 +1,20 @@
+import abis from '../../abis'
+import createUpgradeableLock from './createUpgradeableLock'
+import upgradeLock from './upgradeLock'
+import { parseGetters } from '../../parser'
+
+import v9 from '../v9'
+
+const { createLock, configureUnlock } = v9
+
+const getters = parseGetters(abis.Unlock.v10.abi)
```

Comment:
Hum, so I am not sure this is good enough because `unlock-js` does abstract away a lot of the solidity formats...

Maybe a better approach would be to expose the ethers contract classes in unlock-js letting anyone who needs it just access what they need thru the contracts?